test/files/neg/t8700.check

```diff
@@ -0,0 +1,7 @@
+Bar.scala:2: warning: match may not be exhaustive.
```

Comment:
I have no idea what it says "2" here ... when I run scalac it correctly returns "4".